src/components/common/BoardSection.tsx

```diff
@@ -21,20 +21,20 @@ const ICON = {
   콘텐츠: <FolderIcon fill="#292929" active />,
 };
 
-export default function BoardSection({ title, content, children }: BoardSection) {
+export default function BoardSection({ title, content, mode }: BoardSection) {
   const shouldDisplayControlDate = title === 'My calendar' || title === 'Team calendar';
-  const controlDateMargin = title === 'My calendar' ? 'mr-[6.8rem]' : 'ml-[6.8rem] ';
+  const controlDateMode = mode === 'week' ? 'week' : 'month';
+  const controlDateStyle = mode === 'week' ? 'mr-[111.5rem]' : 'mr-[6.8rem]';
   return (
     <div className="flex h-full flex-col gap-[1.6rem]">
       <div className="relative flex  items-center gap-[0.9rem] whitespace-nowrap text-body1-regular tracking-[-0.036rem] text-gray100">
         {ICON[title]}
-        <span className={`font-rammetto ${controlDateMargin}`}>{title}</span>
+        <span className={`font-rammetto ${controlDateStyle}`}>{title}</span>
         {shouldDisplayControlDate && (
-          <ControlDate className={`text-body4-bold text-gray100`} mode="month" />
+          <ControlDate className={`text-body4-bold text-gray100`} mode={controlDateMode} />
         )}
       </div>
-      <div className="flex-auto">{content || children}</div>
-      {/* content안에 단일 컴포넌트로 넣기 불편한 페이지가 있어서 chilren으로 받을 수 있도록 했습니다.*/}
+      <div className="h-full overflow-auto">{content}</div>
```

Comment:
```suggestion
      <div className="flex-auto">{content || children}</div>
      {/* content안에 단일 컴포넌트로 넣기 불편한 페이지가 있어서 chilren으로 받을 수 있도록 했습니다.*/}
```
기존 작업 부분이 삭제돼서 이렇게 바꿔주세요!